apps/src/studio/Item.js

```diff
@@ -315,10 +340,14 @@ Item.prototype.removeElement = function() {
 };
 
 /**
- * Stop our animations
+ * Retrieve animation frame duration (frames per tick)
  */
-Item.prototype.stopAnimations = function() {
-  this.animation_.stopAnimator();
+Item.prototype.getAnimationFrameDuration = function () {
+  if (this.dir === Direction.NONE) {
+    return this.normalFrameDuration;
```

Comment:
What's special about Direction.NONE that causes us to ignore the Item's custom speed (but still animate it)?